docs/plugins/spectate.rst

```diff
@@ -37,6 +37,7 @@ Usage
     spectate [status]
     spectate toggle
     spectate set <setting> <value> [<subvalue>]
+    spectate toggle <setting>
```

Comment:
could you combine this with the `spectate toggle` above as `spectate toggle [<setting>]`?

Reply:
sure, I just thought it's distinct enough